domains/certificates/Shared/DataContext/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -22,7 +22,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
             NpgsqlModelBuilderExtensions.UseIdentityByDefaultColumns(modelBuilder);
 
-            modelBuilder.Entity("API.ContractService.CertificateIssuingContract", b =>
+            modelBuilder.Entity("DataContext.Models.CertificateIssuingContract", b =>
```

Comment:
Did you test that this works when using dbupdate?

Reply:
I have tested that it works when adding new migrations. Also, the integration tests apply the migrations